src/browser/declarativewebutils.cpp

```diff
@@ -121,8 +121,11 @@ void DeclarativeWebUtils::updateWebEngineSettings()
     SailfishOS::WebEngineSettings *webEngineSettings = SailfishOS::WebEngineSettings::instance();
     SailfishOS::WebEngine *webEngine = SailfishOS::WebEngine::instance();
 
+    MGConfItem userAgent("/apps/sailfish-browser/settings/user_agent");
```

Comment:
I propose `/apps/sailfish-browser/settings/useragent_url` for clarity.

Reply:
or even `/apps/sailfish-browser/settings/useragent_update_url` because this in providing useragent but updates to it.

Reply:
Agree.